index.html

```diff
@@ -47,7 +47,7 @@ <h4 class="connect">
         <div class="item works-1">
             <div class="card-content">
                 <img src="images/works-1.svg" alt="template-image" srcset="">
-                <h3 class="title-1">Tonic</h3>
+                <h3 class="title-1">Tonic .</h3>
```

Comment:
# Changes required 

good job so far, almost there 
#Optional 

- just add more comments to show the beginning of the work section in the index.html

- use more verbs in your commits